unused 'default' field in object causes d.ts generation failure

Build the project and observe the generated InteractionClient.d.ts and note the following errors.  Easy fix is to remove unused `default` field.


```
../chip-matter-wes/packages/matter-node.js/dist/matter/interaction/InteractionClient.d.ts:27:85 - error TS1005: ':' expected.

27     set<T>(_endpointId: number, _clusterId: number, { id, schema, optional, default }: Attribute<T>, _value: T): Promise<void>;
                                                                                       ~

../chip-matter-wes/packages/matter-node.js/dist/matter/interaction/InteractionClient.d.ts:28:112 - error TS1005: ':' expected.

28     subscribe<A extends Attribute<any>>(endpointId: number, clusterId: number, { id, schema, optional, default }: A, listener: (value: AttributeJsType<A>, version: number) => void, minIntervalFloorSeconds: number, maxIntervalCeilingSeconds: number): Promise<void>;```

--- a/packages/matter-node.js/src/matter/interaction/InteractionClient.ts
+++ b/packages/matter-node.js/src/matter/interaction/InteractionClient.ts
@@ -132,14 +132,14 @@ export class InteractionClient {
         });
     }
 
-    async set<T>(_endpointId: number, _clusterId: number, { id: _id, schema: _schema, default: _conformanceValue }: Attribute<T>, _value: T): Promise<void> {
+    async set<T>(_endpointId: number, _clusterId: number, { id: _id, schema: _schema}: Attribute<T>, _value: T): Promise<void> {
         throw new Error("not implemented");
     }
 
     async subscribe<A extends Attribute<any>>(
         endpointId: number,
         clusterId: number,
-        { id, schema, default: _conformanceValue }: A,
+        { id, schema }: A,
         listener: (value: AttributeJsType<A>, version: number) => void,
         minIntervalFloorSeconds: number,
         maxIntervalCeilingSeconds: number,

--- a/packages/matter-node.js/src/matter/interaction/InteractionClient.ts
+++ b/packages/matter-node.js/src/matter/interaction/InteractionClient.ts
@@ -132,7 +132,7 @@ export class InteractionClient {
         });
     }
 
-    async set<T>(_endpointId: number, _clusterId: number, { id: _id, schema: _schema}: Attribute<T>, _value: T): Promise<void> {
+    async set<T>(_endpointId: number, _clusterId: number, { id: _id, schema: _schema }: Attribute<T>, _value: T): Promise<void> {
         throw new Error("not implemented");
     }
 

